modules/cnv-cloning-pvc-of-vm-disk-into-new-datavolume.adoc

```diff
@@ -1,6 +1,9 @@
 // Module included in the following assemblies:
 //
 // * cnv/cnv_users_guide/cnv-cloning-vm-disk-into-new-datavolume.adoc
+// * cnv/cnv_users_guide/cnv-cloning-vm-disk-into-new-datavolume-block.adoc
+//
+// `blockstorage` conditionals are used (declared in the "*-block" assembly) to separate content
```

Comment:
Since it's the same content used across multiple assemblies I figured it was more straightforward to declare it in the assemblies. Unless I can declare multiple contexts in the one ifeval?

Reply:
That's fair. No, the ifeval can have only one argument.